user-interface/src/staff-assignment/StaffAssignmentScreen.scss

```diff
@@ -0,0 +1,13 @@
+.screen-heading {
```

Comment:
in this case wouldn't we want something like this 

`
.my-cases {
   .screen-heading{ ...}
   etc...
}
`
to prevent bleeding of styles. I believe this was discussed before but may or may not apply in this case

Reply:
Yes.